nmmo/core/env.py

```diff
@@ -308,11 +283,7 @@ def step(self, actions: Dict[int, Dict[str, Dict[str, Any]]]):
 
     # Store the observations, since actions reference them
     self.obs = self._compute_observations()
-    gym_obs = {}
-    for a, o in self.obs.items():
```

Comment:
don't we still need this?

Reply:
Previously, this was `gym_obs = {a: o.to_gym() for a,o in self.obs.items()}`, but it was changed to that to add in
```
      if self._task_encoding:
        gym_obs[a]['Task'] = self._encode_goal().get(a,np.zeros(self._task_embedding_size))
```

Currently, I'm not sure about the exact form of the task encoding. Hoping to get some input/specs as we start task conditioned learning very soon